### PR TITLE
[jsk_robot_startup/smach_to_mail] Enable to send INFO tag 

### PR DESCRIPTION
**Pull request by tkmtnt7000**
This PR enables us to send `INFO` tag to email.
`INFO`  is sent as a message to developers.

### CODE REVIEW

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py [hunk 1]
```diff
@@ -172,16 +184,21 @@ def _send_mail(self, subject, state_list):
         self.pub_email.publish(email_msg)
 
     def _send_twitter(self, subject, state_list):
-        text = ""
+        text = u""
         if subject:
+            if isinstance(subject, bytes):
```
Comment (k-okada) [inline jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py:189]: Could you add commends like `# In python2, 日本語  is bytes? and ......, so we need to .....`

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py [hunk 2]
```diff
@@ -82,6 +82,11 @@ def _status_cb(self, msg):
             rospy.logwarn("smach does not have DESCRIPTION, see https://github.com/jsk-ros-pkg/jsk_robot/tree/master/jsk_robot_common/jsk_robot_startup#smach_to_mailpy for more info")
         if 'IMAGE' in local_data_str and local_data_str['IMAGE']:
             rospy.loginfo("- image_str -> {}".format(local_data_str['IMAGE'][:64]))
+        if 'INFO' in local_data_str:
+            rospy.loginfo("- description_str -> {}".format(local_data_str['INFO']))
```
[the remark this answers is not in the document]
Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py]: Thanks. I fixed it in [e89d5bd](https://github.com/jsk-ros-pkg/jsk_robot/pull/1584/commits/e89d5bd3d9c10ebb996c40c08d9fa8a8db673267)

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py [hunk 3]
```diff
@@ -157,6 +162,13 @@ def _send_mail(self, subject, state_list):
                 image.img_data = x['IMAGE']
                 email_msg.body.append(image)
                 email_msg.body.append(changeline)
+        for x in state_list:
```
[the remark this answers is not in the document]
Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py:172]: Thank you. I added in [1f8d91e](https://github.com/jsk-ros-pkg/jsk_robot/pull/1584/commits/1f8d91e542edfcea661bca1b92cac6e8b11b55c2)

File: jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py (hunk 1 above)
Comment (tkmtnt7000) [reply jsk_robot_common/jsk_robot_startup/scripts/smach_to_mail.py:197]: Yes. I added comment in [85d6a77](https://github.com/jsk-ros-pkg/jsk_robot/pull/1584/commits/85d6a77d9afc0ba4b5c67de56347f468d459724e)

**Pull request merged**